tests/test_oom_logger.py

```diff
@@ -86,7 +86,11 @@ def sys_stdin_without_process_name():
 def docker_inspect():
     return {
         "Config": {
-            "Env": ["PAASTA_SERVICE=fake_service", "PAASTA_INSTANCE=fake_instance"]
+            "Env": [
+                "PAASTA_SERVICE=fake_service",
+                "PAASTA_INSTANCE=fake_instance",
+                "PAASTA_HOST=dev42-uswest1adevc-64e7207e-f738-4236-b0c5-8f4fc5e0adbd",
```

Comment:
Are you sure about this example? I thought the new PAASTA_HOST variable would only have the hostname in it

Reply:
I upgraded PAASTA_HOST to use box's hostname

Reply:
Ok, oops that was a misunderstanding of that change, in that case. I am just going to grab `HOSTNAME` from inside the container, which is currently what gets logged in the logs we need to join to. hostname from outside the container is the fallback, which doesn't have the mesos task id information that is logged from the service (to identify which is the OOM'ing instance on a host).